lib/toml-parser.js

```diff
@@ -162,9 +162,14 @@ function isList (obj) {
 }
 
 // in an eval, to let bundlers not slurp in a util proxy
-const utilInspect = eval(`require('util').inspect`)
-/* istanbul ignore next */
```

Comment:
Deleting the instanbul directive results in tests failing due to 172 not having coverage for the alternative.